.dockerignore

```diff
@@ -1,5 +1,9 @@
 .git
-.gradle
 .gradletasknamecache
 *.jar
 **/build/
+
+# Should exclude the whole .gradle and un-exclude .gradle/tomcat, but the Gradle plugin doesn't support it
+.gradle/4*
```

Comment:
here a version range could be a good for future proofing.
```gitignore
.gradle/[0-9]*
```

ref: https://docs.docker.com/engine/reference/builder/#dockerignore-file
ref: https://golang.org/pkg/path/filepath/#Match

Reply:
Not a bad idea.  What about something like...

```
.gradle/[0-9A-Za-su-z]*
```

which would let only `.gradle/t*` through (barring special characters).

Reply:
An inverted range then?
```
.gradle/[^t]*
```